src/poprox_recommender/components/joiners/concat.py

```diff
@@ -0,0 +1,9 @@
+from poprox_concepts import ArticleSet
+
+
+class Concatenate:
+    def __call__(self, candidates1: ArticleSet, candidates2: ArticleSet) -> ArticleSet:
+        reverse_articles = (candidates1.articles + candidates2.articles)[::-1]
+        articles = {article.article_id: article for article in reverse_articles}
```

Comment:
Might this be more readable (less dependent on list indexing magic) if it is:

```python
articles = {article.article_id: article for article in reversed(candidates1.articles + candidates2.articles)}
```

This would also have advantage of reducing one list allocation, but that isn't a sufficient reason to do it.

Reply:
Mostly thinking that explicit use of `reversed` may be easier to understand at first look for less-experienced Python programmers — it does exactly what it says.

Reply:
It's interesting: I typically don't use nearly as much indexing magic on my own as the students seem to reach for